src/Functions/formatDateTime.cpp

```diff
@@ -740,8 +1261,21 @@ struct NameFromUnixTime
     static constexpr auto name = "fromUnixTimestamp";
 };
 
-using FunctionFormatDateTime = FunctionFormatDateTimeImpl<NameFormatDateTime, false>;
-using FunctionFromUnixTimestamp = FunctionFormatDateTimeImpl<NameFromUnixTime, true>;
+struct NameFormatDateTimeInJodaSyntax
+{
+    static constexpr auto name = "formatDateTimeInJodaSyntax";
+};
+
+struct NameFromUnixTimeInJodaSyntax
+{
+    static constexpr auto name = "fromUnixTimestampInJodaSyntax";
+};
+
+
+using FunctionFormatDateTime = FunctionFormatDateTimeImpl<NameFormatDateTime, FormatDateTimeTraits::SupportInteger::No, FormatDateTimeTraits::FormatSyntax::MySql>;
+using FunctionFromUnixTimestamp = FunctionFormatDateTimeImpl<NameFromUnixTime, FormatDateTimeTraits::SupportInteger::No, FormatDateTimeTraits::FormatSyntax::MySql>;
```

Comment:
I am pretty sure that in l. 1276, we need `SupportInteger::Yes` instead of `SupportInteger::No`.